app/views/teachers/_form.html.erb

```diff
@@ -91,14 +91,14 @@
     var status_val = $( "#teacher_status" ).val();
 
     if (status_val == 3) {
```

Comment:
oh let's clean this up. :) 

Let's add a new text field on the form which looks something like:
```html
<strong id='teals-volunteer-more-info' class='hidden'>Please include the name of the your host teacher</strong>
```

And then the JavaScript can do `$('#eals-volunteer-more-info').show()` or `.hide()`

Reply:
Also, let's delete the old lines of code that are commented out.